crypto.c

```diff
@@ -24,14 +24,17 @@
 #else
 #define assert(x)
 #endif
+#include <inttypes.h>
 
 #include "global.h"
 #include "dtls_debug.h"
 #include "numeric.h"
 #include "dtls.h"
 #include "crypto.h"
 #include "ccm.h"
-#include "ecc/micro-ecc/uECC.h"
+#ifdef DTLS_ECC
+#include "ext/micro-ecc/uECC.h"
```

Comment:
"ext/micro-ecc" is part of the include path, maybe it's simpler to use "uECC.h"?

Reply:
It is my preference to always include the package information path in the #include statement, to prevent name clashes (unlikely with `uECC.h`, but with possible with names like `types.h` etc.).

So, I would recommend that the include path includes `ext/`, and use `#include <micro-ecc/uECC.h>`.

Reply:
The pain is, that "micro-ecc/uECC.c" uses: 

```
#include "uECC.h"
#include "uECC_vli.h"
```

and "micro-ecc/uECC_vli.h" uses:

```
#include "uECC.h"
#include "types.h"
```

Reply:
Anything that is getting built external to micro-ecc will need the equivalent of "micro-ecc/uECC.h" to get at the function definitions.  For the actual micro-ecc code compilation, I assume that it is done in the micro-ecc/ directory, in which case just "uECCH.h" without a path is fine.
````
If you declare #include "" , the compiler first searches in current directory of source file and if not found,
continues to search in any included -I directories and then the default include directories.

If you declare #include <> , the compiler searches directly in any included -I directories and then the default
include directories.

$ cpp -v shows the include search directory order.
````

Reply:
OK. I've checked it and at least the cmake build doesn't need to extend the include path with "ext/micro-ecc".

Reply:
> "ext/micro-ecc" is part of the include path,

I mixed it up, it's only part of "FILES".